### DISCUSSION TITLE
layouts/_default/_markup/render-link.html doesn't work

### DISCUSSION THREAD

**Discussion by ckamps** [gethinode/hinode#825] [General]
I'm trying to make use of the Hugo feature with Goldmark to force http* links to open in a new tab, but the common approach does not work.

The Hugo doc states "A custom render hook, even when provided by a theme or module, will override the embedded render hook regardless of the configuration setting above."

Is there something in the Hinode theme that inhibits this common approach from working?

* Hinode v0.22.2
* Hugo 0.122.0

`layouts/_default/_markup/render-link.html`

```
<a href="{{ .Destination | safeURL }}"{{ with .Title}} title="{{ . }}"{{ end }}{{ if strings.HasPrefix .Destination "http" }} target="_blank"{{ end }}>{{ .Text }}</a>
```

**Comment by markdumay** [comment]
The embedded render link hook is [introduced by Hugo v0.123.0](https://gohugo.io/render-hooks/links/#default), so your version v0.122.0 is not compatible.

Hinode offers a similar feature that you might find helpful, see the [link shortcode ](https://gethinode.com/docs/components/link/) documentation. I've added feature request #826 to adapt the new render hook.

As a final note, Hugo v0.123.x introduced a few bugs that are being ironed out by its maintainers. You can use the latest [Hinode beta](https://github.com/gethinode/hinode/releases/tag/v0.23.0-beta9), which is compatible with Hugo v0.123.7.

**Comment by markdumay** [comment]
I have released [Hinode v0.23.0-rc.1](https://github.com/gethinode/hinode/releases/tag/v0.23.0-rc.1), which adds support for Hugo v0.123.8.
